src/tls.c

```diff
@@ -11918,6 +11920,7 @@ int TLSX_Parse(WOLFSSL* ssl, const byte* input, word16 length, byte msgType,
 #if defined(WOLFSSL_TLS13) && (defined(HAVE_SESSION_TICKET) || !defined(NO_PSK))
     int pskDone = 0;
 #endif
+    byte seenType[SEMAPHORE_SIZE] = {0};    /* Seen known extensions. */
```

Comment:
Prefer XMEMSET vs ` = {0}` initializer (for portability).

Reply:
Fixed